rapids/rapids.sh

```diff
@@ -30,7 +30,7 @@ readonly RUN_WORKER_ON_MASTER=$(get_metadata_attribute 'dask-cuda-worker-on-mast
 readonly DEFAULT_CUDA_VERSION="10.2"
```

Comment:
I wonder if we should select cuda 11 as default. I am making the changes in GPU driver as well, there is an on-going pull request and I need cover a test failure. But ampere should reach dataproc soon.

Reply:
Once you test cuda 11 on Dataproc and confirm it works we should update.

Reply:
I have run a simple GPU job with cuda 11 and it execute on GPU
```
== Physical Plan ==                                                   
*(3) GpuColumnarToRow false                                           
+- GpuShuffledHashJoin [_1#0L], [_1#2L], Inner, BuildRight, false     
   :- GpuCoalesceBatches TargetSize(2147483647)                       
   :  +- GpuColumnarExchange gpuhashpartitioning(_1#0L, 48), [id=#230]
   :     +- GpuCoalesceBatches TargetSize(2147483647)                 
   :        +- GpuFilter gpuisnotnull(_1#0L)                          
   :           +- GpuRowToColumnar TargetSize(2147483647)             
   :              +- *(1) Scan ExistingRDD[_1#0L]                     
   +- GpuCoalesceBatches RequireSingleBatch                           
      +- GpuColumnarExchange gpuhashpartitioning(_1#2L, 48), [id=#236]
         +- GpuCoalesceBatches TargetSize(2147483647)                 
            +- GpuFilter gpuisnotnull(_1#2L)                          
               +- GpuRowToColumnar TargetSize(2147483647)             
                  +- *(2) Scan ExistingRDD[_1#2L]
```